src/app/shared/components/lmt-autocomplete/lmt-autocomplete.component.ts

```diff
@@ -99,6 +112,23 @@ export class LmtAutocompleteComponent {
     );
   }
 
+  onChange: any = () => {}
```

Comment:
// function to call when selected items change
onChange = ( selectedItems: any[] ) => {};